extensions/control-plane/api/management-api/management-api-configuration/src/main/java/org/eclipse/edc/connector/api/management/configuration/ManagementApiConfigurationExtension.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.api.management.configuration;
+
+import org.eclipse.edc.api.auth.spi.AuthenticationRequestFilter;
+import org.eclipse.edc.api.auth.spi.AuthenticationService;
+import org.eclipse.edc.runtime.metamodel.annotation.Extension;
+import org.eclipse.edc.runtime.metamodel.annotation.Inject;
+import org.eclipse.edc.runtime.metamodel.annotation.Provides;
+import org.eclipse.edc.spi.system.ServiceExtension;
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.web.spi.WebServer;
+import org.eclipse.edc.web.spi.WebService;
+import org.eclipse.edc.web.spi.configuration.WebServiceConfigurer;
+import org.eclipse.edc.web.spi.configuration.WebServiceSettings;
+
+import static java.lang.String.format;
+
+/**
+ * Tells all the Management API controllers under which context alias they need to register their resources: either `default` or `management`
+ *
+ */
+@Provides(ManagementApiConfiguration.class)
+@Extension(value = ManagementApiConfigurationExtension.NAME)
+public class ManagementApiConfigurationExtension implements ServiceExtension {
+
+    public static final String NAME = "Management API configuration";
+
+    private static final String DEPRECATED_MANAGEMENT_SETTINGS_GROUP_SUFFIX = "data";
+    private static final String DATA_MANAGEMENT_CONTEXT_ALIAS = "management";
+    private static final int DEFAULT_DATA_MANAGEMENT_API_PORT = 8181;
+    private static final String DEPRECATED_DEFAULT_DATA_MANAGEMENT_API_CONTEXT_PATH = "/api";
+    private static final String DEFAULT_DATA_MANAGEMENT_API_CONTEXT_PATH = "/api/v1/management";
```

Comment:
the `DATA` prefix could also be removed from the constants

Reply:
it's actually used to handle the deprecation: `web.http.data` will be still usable for some time printing a warning, so the users will have the time to switch to the new settings group.